validator/sawtooth_validator/journal/chain.py

```diff
@@ -505,6 +505,37 @@ def _verify_block(self, blkw):
         self._blocks_processing[blkw.block.header_signature] = validator
         self._executor.submit(validator.run)
 
+    def _remove_block_and_verify_descendants(self, parent_block):
```

Comment:
It was initially named `_verify_descendants()` and I went back and forth between names, and settled on `_remove_...()` because it was explicit about the removing part.  I can change back.